### PR TITLE
Bump gclib.

### PR DESCRIPTION
**Pull request by cbrueffer**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/gclib/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+
 package:
-  name: gclib
-  version: "0.0.1"
+  name: {{ name }}
+  version: {{ version }}
 
 build:
-    number: 4
+    number: 5
```
Comment (bgruening) [inline recipes/gclib/meta.yaml]: I'm not sure increase the build number is correct here. We should increase the actual version.

**Comment by cbrueffer** [comment]
gclib is not actually versioned in any way, it's more of a continuous development (at least as I can see). Do you see any alternative in this case?

**Comment by bgruening** [comment]
we can simply increase the 0.1 version by 0.1.1 and increase the last number

**Comment by johanneskoester** [comment]
We  have made changes to our build system (migrated from Travis CI to  Circle CI). If you would like to continue working on this PR, please  reopen this PR and update with latest upstream/master.

**Pull request closed without merging**